### PR TITLE
Tryingtodoshopthings

### PR DESCRIPTION
**Pull request by KatBirk**
Now towers are placeable as separate entities, and cost is subtracted.

### CODE REVIEW

File: GameEngine/src/main/java/dk/sdu/se/f23/InVasion/main/Game.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void create() {
             plugin.onEnable(gameData, world);
             System.out.println(plugin.getClass().getName() + " loaded");
         }
-        gameData.setPlayerMoney(5000);
+        gameData.setPlayerMoney(200);
```
Comment (Darchie4) [inline GameEngine/src/main/java/dk/sdu/se/f23/InVasion/main/Game.java:55]: Should be 0, but okay...

**Review by Darchie4 — APPROVED** [review]
Looks good

File: GameEngine/src/main/java/dk/sdu/se/f23/InVasion/main/Game.java (hunk 1 above)
Comment (Crazi12345) [reply GameEngine/src/main/java/dk/sdu/se/f23/InVasion/main/Game.java:55]: disagree
200 > 0
NO more should be said

**Review by Crazi12345 — APPROVED** [review]
Very good

**Pull request merged**